docs/40_smart-contract-guides/92_read-only-queries.md

```diff
@@ -0,0 +1,337 @@
+---
+content_title: "Read-Only Queries"
+link_text: "Read-Only Queries"
+---
+
+
+External clients were limited to accessing chain state data through the get_tables APIs for chainbase or KV tables or through various history solutions. In version 2.2 of the there is now an API to invoke contract-defined read-only queries and return the results. Smart contracts can be written to include actions which are read-only queries using the `read-only` attribute. This tutorial will show you how to create a simple smart contract with a read-only query, and how to call this query from the cleos CLI. 
+
+Once the tutorial is completed you should be able to create and call a read-only query.
+
+## Before you begin
+This tutorial requires the following:
+
+* The EOSIO platform software, version 2.2. or greater, click on this link for instructions on [installing EOSIO binaries](https://developers.eos.io/manuals/eos/latest/install/install-prebuilt-binaries)
```

Comment:
Also call out the CDT binaries, with a corresponding link to instructions.

Reply:
updated - to say cdt 1.9 and greater - Jonathan can you confirm the version number

Reply:
Product is advocating for a 1.8.1 release number.  I don't believe we have resolved the question yet.

Reply:
put in 1.8.1 for time being easy to update once someone decides what the version number is